packages/site/src/components/Navigation/PrimarySidebar.tsx

```diff
@@ -189,7 +189,10 @@ export const PrimarySidebar = ({
               aria-label="Table of Contents"
               className="flex-grow overflow-y-hidden transition-opacity ml-3 xl:ml-0 mr-3 max-w-[350px]"
             >
-              <Toc headings={headings} />
+              <Toc
+                headings={headings}
+                open_urls_in_new_tab={config?.options?.open_toc_urls_in_new_tab}
```

Comment:
One other thought - would we ever want to have different values for this option for different toc entries...?

If so, we couldn't do this as a single, site-wide option; it would have to be an optional field on each toc entry:

```
toc:
  - file: index.md
  - title: Example
    url: https://example.com
    same_tab: true
```
or something...

I suspect this toc-entry-level granularity is unnecessary at this point, but maybe a limitation to keep in mind.

Reply:
This has now been implemented: `open_in_same_tab`